src/plugins/intel_cpu/src/emitters/tpp/x64/jit_eltwise_emitters.cpp

```diff
@@ -0,0 +1,153 @@
+// Copyright (C) 2020-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "jit_eltwise_emitters.hpp"
+
+#include <cpu/x64/jit_generator.hpp>
+
+#include "snippets/lowered/port_connector.hpp"
+#include "snippets/lowered/port_descriptor.hpp"
+#include "transformations/snippets/tpp/op/eltwise.hpp"
+#include "snippets/lowered/port_descriptor.hpp"
+
+using namespace Xbyak;
+using namespace dnnl::impl;
+using namespace dnnl::impl::cpu::x64;
+
+namespace ov {
+namespace intel_cpu {
+using jit_generator = dnnl::impl::cpu::x64::jit_generator;
+using cpu_isa_t = dnnl::impl::cpu::x64::cpu_isa_t;
+using ExpressionPtr = ov::snippets::lowered::ExpressionPtr;
+using ExpressionPort = snippets::lowered::ExpressionPort;
```

Comment:
Could you remove these lines as well please?

Reply:
I can't, we need them. I removed redundant namespaces in the constructors, please take a look.